README.md

```diff
@@ -12,13 +13,46 @@ The Application is made with:
 * Python, Django & Django-REST-Framework
 * Postgres, Tests
 
-To start run: ```python manage.py runserver```
+Set up instructions:
+
+1. Assuming you have postgres installed, create project database:
+    ```
+        sudo su - postgres  
+        psql  
+        CREATE DATABASE djangoapp;  
+        CREATE USER coder WITH PASSWORD 'coderpwd';  
+        \q  
+        exit  
+    ```
+
+2. In the folder **django-app** make a virtualenv called myenv.  
+    ```python3 -m venv myenv```
+
+3. Activate virtual environment (include full path if required).  
+    ```source djangoapp/myenv/bin/activate```
+
+4. Install requirements of the project.  
+    ```pip install -r requirements.txt```
+
+5. Create and apply migrations.  
+    ```
+    python manage.py makemigrations  
```

Comment:
According to Django's docs on https://docs.djangoproject.com/en/3.1/topics/migrations/

> makemigrations, which is responsible for creating new migrations based on the changes you have made to your models.

When setting up the dev environment there's no point running this because you are not meant to change any models yet. We only want to apply them into our local DB, which we do below.

Reply:
It's true it is not necessary here.